Assets/Scripts/Nathan/Animator/AnimatorMovement.cs

```diff
@@ -4,11 +4,15 @@ public class AnimatorMovement : MonoBehaviour
 {
     public GameObject playerGhost;
 
+    const float walkingSpinSpeed = 8000;
+    const float walkingSpeed = 15;
+
     private Animator animator;
     private Rigidbody2D playerRigidBody;
+    private Rigidbody2D playerGhostRigidBody;
     private Vector2 grabPoint;
     private Vector2 grabPointOffset;
-    private bool grabbedChanged;
+    private bool grabFlag;
     //private bool grabbed;    <--- this is now be an animator controlled variable instead.
 
```

Comment:
Add `private static readonly int Grabbed = Animator.StringToHash("Grabbed");` so that we can avoid string-based lookup

Reply:
Does not work.